devoxx-2017-be/cinema/src/main/java/org/tweetwallfx/devoxx2017be/dataprovider/SessionData.java

```diff
@@ -48,7 +48,7 @@
                 .filter(slot -> null != slot.getTalk())
                 .filter(slot -> slot.getTalk().getTalkType() != "BOF (Bird of a Feather)")
                 .filter(slot -> OffsetTime.parse(slot.getToTime() + "Z").isAfter(now))
-                .filter(slot -> OffsetTime.parse(slot.getFromTime() + "Z").isBefore(now.plusMinutes(60)))
+                .filter(slot -> !OffsetTime.parse(slot.getFromTime() + "Z").isBefore(now.plusMinutes(120)))
```

Comment:
I believe it is wrong to exclude all talks that start within the next 2 hours. Please verify this is really intended